code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -358,15 +364,15 @@
 //////////////////////////SPACEWIND/////////////////////////////
 
 /turf/open/proc/consider_pressure_difference(turf/target_turf, difference)
+	if(difference < pressure_difference)
+		return
+
 	SSair.high_pressure_delta |= src
```

Comment:
I think? there was a reason for this, it doesn't come immediately to mind. bit uneasy about this